src/include/main/db_main.h

```diff
@@ -424,6 +425,12 @@ class DBMain {
         messenger_layer = std::make_unique<MessengerLayer>(common::ManagedPointer(thread_registry), messenger_port_,
                                                            messenger_identity_);
       }
+      std::unique_ptr<model::ModelServerManager> model_server_manager = DISABLED;
+      if (pilot_enable_) {
```

Comment:
rename all instances of Pilot to ModelServerManager? model_server_manager_enable_, ...

Reply:
Is @rabbit721 Pilot gonna be something the MSM should depend on or vice versa, or they are independent components of the system?
I saw in her demo today that she had a `pilot_enable` flag, not sure if we want to use that to enable MSM as well?

Reply:
From Lin's previous messages on slack, Pilot should be able to invoke the mini_trainer and do inference of models through MSM. So I think we would need MSM enabled when the Pilot is enabled.

Reply:
The pilot is dependent on the MSM since the pilot needs to perform inference through MSM. The training might be separate though. We might directly invoke the trainers once at system start time or just directly import some pre-trained models (by specifying the model path during inference).